WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -1014,6 +1014,10 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
     func gutenbergDidRequestSetBlockTypeImpressions(_ impressions: [String: Int]) -> Void {
         gutenbergSettings.blockTypeImpressions = impressions
     }
+
+    func gutenbergDidRequestSendEventToHost(_ eventName: String, properties: [AnyHashable: Any]) -> Void {
+        WPAnalytics.trackBlockEditorEvent(eventName, properties: properties, blog: post.blog)
+    }
```

Comment:
This function is another candidate location for the event name string to `WPAnalyticEvent` conversion. The argument for this location would be an attempt to avoid "leaking" JavaScript-related patterns into the WPiOS app, i.e. keep the idea of string event names out of the `WPAnalyticsEvent` file.